src/audio/dai.c

```diff
@@ -701,6 +702,7 @@ static int dai_comp_trigger_internal(struct comp_dev *dev, int cmd)
 		comp_dbg(dev, "dai_comp_trigger_internal(), PAUSE");
 		ret = dma_pause(dd->chan);
 		dai_trigger(dd->dai, cmd, dev->direction);
+		__attribute__((fallthrough));
```

Comment:
This one looks like it should be a ```break```

Reply:
ping

Reply:
Will fix, but what about the other, this fix breaks ccc according to CI, should I just make this conditional based on if clang is compiling?

Reply:
The other one looked fine with __attribute__((fallthrough) as this should also work with gcc ?

Reply:
But it breaks xcc according to CI.

Reply:
Ah, didn't read that right - we probably need to add this to compiler_attr.h and define some macros for this.